src/deprecation.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2017 The Zcash developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef ZCASH_DEPRECATION_H
+#define ZCASH_DEPRECATION_H
+
+// Deprecation policy is 4th third-Tuesday after a release
+static const int APPROX_RELEASE_HEIGHT = 115000;
```

Comment:
It's important to me that this has a very high chance of occuring after the release day. Add a comment with release date and back-of-the-envelope calculations showing some grace period.